### DISCUSSION TITLE
求助是否需要抽代理人朱鸢

### DISCUSSION THREAD

**Discussion by 624140659** [share121/inter-knot#1061] [其他]
早上起来一个十连抽直接出的双金专武，不过歪了一个。想着补一个代理人，直接歪了，现在如果强娶青衣大概率就拿不到了，我想着给鲨鱼过渡一下得了。

**Comment by match-fire** [comment]
别抽！没有游戏体验感！输出只能长A，一会怪就死了（
毕竟是第二个限五，又是警员，很难不想想隔壁牢景的现状
当然有能力还是抽一个，配队也很简单，带个妮可，安比就行

**Comment by Friskdrink** [comment]
看情况，目前版本主c较缺，如果有鲨鱼妹和11号可以不抽，但如果只有一个c，推荐抽【毕竟常驻深渊第六层后开始需要二队

**Reply by 624140659** [reply]
手里一队鲨鱼二队物理异常队。给鲨鱼用还意外的不错

**Comment by DonQuixote084** [comment]
可恶的欧狗来炫耀了，我这就把你

**Comment by 624140659** [comment]
还真不是炫耀，没有代理人有专武我玩集贸
